reco_utils/nni/svd_training.py

```diff
@@ -0,0 +1,150 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
there is another svd_training for azureml, do you think it is possible to have a common one?

Reply:
Not really, because Hyperdrive and NNI have differences and there are differences in how they parse arguments, the arguments are not the same in fact, reporting of the metrics differs etc. Even if it were possible to refactor some common code, it would be confusing to the developer.

Reply:
ok